src/pages/Graph/GraphPage.tsx

```diff
@@ -158,6 +158,7 @@ export class GraphPage extends React.Component<GraphPageProps, GraphPageState> {
     const workload = props.match.params.workload;
     const workloadOk = workload && workload !== UNKNOWN && workload !== 'undefined';
     if (!appOk && !namespaceOk && !serviceOk && !workloadOk) {
+      // @ts-ignore
```

Comment:
I'm curious why this ts-ignore is necessary, whereas there's no change in tsconfig.json ?
If unsure, maybe after that PR we should have a global "ts-ignore-cleaning" task, wdyt?

Reply:
+1 to have a followup tasks to review the "ts-ignore", perhaps we can clean the code in more places.
I guess this one was suggested by @mtho11 in the context to resolve the blocker we have.

I guess we can consolidate this and clean the ts-ignore in single / separted task.